src/__tests__/Card.js

```diff
@@ -90,4 +90,10 @@ describe('Card', () => {
       '0 0 2px 0 rgba(0,0,0,.08),0 2px 8px 0 rgba(0,0,0,.16)'
     )
   })
+
+  test('renders border 0 without warning', () => {
+    const json = renderer.create(<Card borderWidth={0} />).toJSON()
+    expect(json).toMatchSnapshot()
+    expect(json).toHaveStyleRule('border', '0')
```

Comment:
This isn't actually testing whether the console warning is being fired or not. I think we have some other places where we use `jest.fn()` to test prop types

Reply:
Are you planning to update the test here like you did in #416?

Reply:
yes, sorry, just been busy

Reply:
pushed a similar use of console.error.mock.calls